apps/studio/src/features/settings/SettingsSidenav/SettingsSidenav.tsx

```diff
@@ -1,18 +1,74 @@
 import type { UseDisclosureReturn } from "@chakra-ui/react"
+import { useRouter } from "next/router"
+import { VStack } from "@chakra-ui/react"
+import { IconType } from "react-icons"
+import { BiWrench } from "react-icons/bi"
 
 import { CmsCollapsibleSidenav } from "~/components/CmsSidebar/CmsCollapsibleSidenav"
+import { siteSchema } from "~/features/editing-experience/schema"
+import { useQueryParse } from "~/hooks/useQueryParse"
+import { HeaderRow } from "./components"
+import { SettingsItem } from "./components/SettingsItem"
 
 interface SettingsSidenavProps {
   onSidenavClose: UseDisclosureReturn["onClose"]
 }
 
+interface SideNavItem {
+  header: {
+    label: string
+    icon: IconType
+  }
+  items: {
+    label: string
+    href: string
+  }[]
+}
+
 export const SettingsSidenav = ({ onSidenavClose }: SettingsSidenavProps) => {
+  const { siteId } = useQueryParse(siteSchema)
+  const router = useRouter()
+
+  const SIDENAV_ITEMS: SideNavItem[] = [
+    {
+      header: { label: "General", icon: BiWrench },
+      items: [
+        { label: "Name and agency", href: `/sites/${siteId}/settings/agency` },
+        {
+          label: "Notification banner",
+          href: `/sites/${siteId}/settings/notification`,
+        },
+      ],
+    },
+  ]
+
   return (
     <CmsCollapsibleSidenav
       title="Site settings"
       onSidenavClose={onSidenavClose}
     >
-      <p>Content in sidenav</p>
+      {SIDENAV_ITEMS.map(({ header, items }) => {
+        return (
+          <VStack align="start">
+            <HeaderRow {...header} />
+            <VStack
+              borderLeft="1px solid"
+              borderColor="base.divider.medium"
```

Comment:
super nit: this more of design, I thought it would make a bit more sense if the border is also `interaction.main.default` if active, not just the text

Reply:
cc @sehyunidaaa but i didn't do